lib/SPIRV/libSPIRV/SPIRVUtil.h

```diff
@@ -40,6 +40,8 @@
 #ifndef SPIRV_LIBSPIRV_SPIRVUTIL_H
 #define SPIRV_LIBSPIRV_SPIRVUTIL_H
 
+#include "SPIRVLLVMUtil.h" // LLVM-specific utilities
```

Comment:
As this transitively includes a lot of LLVM headers, preferably include this only where you really need it, so in SPIRVWriter.cpp.  This helps keeping llvm-spirv compilation times reasonable.

Also, the comment is not really needed in my opinion.

Reply:
Fair enough, done.